docs/config/cluster.rst

```diff
@@ -105,7 +105,7 @@ Collecting stats
   common use case is to include only jobs that took a certain amount of time to
   execute::
 
-    cr> SET GLOBAL "stats.jobs_log_filter" = 'ended - started > 100';
+    cr> SET GLOBAL "stats.jobs_log_filter" = $$ended - started > '5 minutes'::interval$$;
```

Comment:
minor: considering if we should use the standard SQL escaping instead:
```
'ended - started > ''5 minutes''::interval'
```

Reply:
I don't see what's better about that? Either way works